scripts/west_commands/runners/jlink.py

```diff
@@ -138,8 +108,34 @@ def dev_id_help(cls) -> str:
     def tool_opt_help(cls) -> str:
         return "Additional options for JLink Commander, e.g. '-autoconnect 1'"
 
+    @staticmethod
+    def default_jlink():
+        global DEFAULT_JLINK_EXE
+
+        if sys.platform == 'win32':
+            # JLink.exe can collide with the JDK executable of the same name
+            # Locate the executable using the registry
+            try:
+                import winreg
+                # Note that when multiple JLink versions are installed on the
```

Comment:
@JordanYates this is a bit of a difference compared to your approach, let me know what you think

Reply:
Most recently installed is preferable to the previous behavior of latest version IMO